File: csL2/Domain/Participant.cs
namespace lab2_mpp.domain;

public class Participant
{
    public String Name { get; set; }
    public int Age { get; set; }
    public int NrProbe { get; set; }

    public Participant(String name, int age, int nrProbe)
    {
        Name = name;
        Age = age;
        NrProbe = nrProbe;
    }
    
    public override string ToString()
    =>$"Participant {{ name: {Name}, age: {Age}, nrProbe: {NrProbe}}}";
}

File: Project/Model/src/main/java/ro/mpp2024/domain/Participant.java
package ro.mpp2024.domain;

import java.io.Serializable;

public class Participant extends Entity<Integer> {
    private String name;
    private int age;
    private int nrProbe;


    public Participant(String name, int age) {
        this.name = name;
        this.age = age;

    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public int getAge() {
        return age;
    }
    public void setAge(int age) {
        this.age = age;
    }
    public int getNrProbe() {
        return nrProbe;
    }
    public void setNrProbe(int nrProbe) {
        this.nrProbe = nrProbe;
    }
    @Override
    public String toString() {
        return "Participant{id=" + getId() + ", name=" + name + ", age=" + age + "}";
    }
}
